x-pack/dockerlogbeat/docs/configuration.asciidoc

```diff
@@ -76,6 +76,11 @@ indices. For example: +"dockerlogs-%{+yyyy.MM.dd}"+.
 
 3+|*Advanced:*
 
+|`name`
+|`testbeat`
+| A custom value that will be inserted into the document as `agent.name`.
+If not set, it will be the same value as `agent.type`
```

Comment:
This might be a little confusing for Beats users. Libbeat also provides a 'global' `name` setting, but defaults to the hostname if `name` is not configured. This is done, because `name` is used by users to identify a single instance (e.g. if multiple beats are run on a single host). Do we want to default to the hostname instead of `agent.type` as well?